lib/common/common_search_bar.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class SearchBar extends StatelessWidget {
+  const SearchBar({super.key, this.onChanged});
+
+  final ValueChanged?
+      onChanged; //TODO(LW): change this to required one functionality begins
+
+  @override
+  Widget build(BuildContext context) {
+    return TextField(
+      decoration: InputDecoration(
+        prefixIcon: const Icon(Icons.search),
+        border: OutlineInputBorder(
+          borderRadius: BorderRadius.circular(25),
+          borderSide: BorderSide.none,
+        ),
+        filled: true,
+        fillColor: Colors.grey[
+            300], //TODO(LJ): you may want to change this... couldn't find it in theme
```

Comment:
@lyjohnson let me know what color this should be/if this is fine... couldn't find in theme